ui/src/views/PortForward.vue

```diff
@@ -952,6 +959,9 @@ export default {
     savePF() {
       var context = this;
 
+      // Remove empty lines and whitespace at begin and end of Allow
+      var CleanAllow = ((context.newPf.Allow.split("\n")).filter(e => String(e).trim())).map(str => str.trim());
```

Comment:
this can be simplified
```suggestion
      var CleanAllow = context.newPf.Allow.split("\n").map(str => String(str).trim());
```

Reply:
I need to filter an empty value element but also to trim whitespace at the end/beginning of an array element

Reply:
Yes, my fault. Ignore it.